install/easy-install

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# FOSSology Easy Install script
+# Copyright Siemens AG 2017
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Description: Easy install script for fossology. Goes through the steps on the Fossology wiki
+set -e
+cd "$(dirname $(readlink -f "${BASH_SOURCE[0]}"))"/..
+sudo utils/fo-cleanold
+make clean
+sudo utils/fo-installdeps
+make
+sudo make install && sudo /usr/local/lib/fossology/fo-postinstall
+sudo cp install/src-install-apache-example.conf /etc/apache2/sites-available/fossology.conf
+sudo cp install/src-install-apache-example.conf /etc/apache2/sites-enabled/fossology.conf
```

Comment:
I don't know if that's the right way, because a2ensite only works on debian based systems.